<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton  {
    background-color: rgb(138, 255, 43);
    color:white;
    border-style: outset;
    border-width: 5px;
    border-radius: 10px;
    border-color: rgb(57, 255, 2)
}

QPushButton:pressed {
     background-color:rgb(49, 49, 49);
     border-color:rgb(49, 49, 49);
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton  {
    background-color: rgb(119, 137, 255);
    color:white;
    border-style: outset;
    border-width: 5px;
    border-radius: 10px;
    border-color: rgb(6, 85, 255)
}

QPushButton:pressed {
     background-color:rgb(49, 49, 49);
     border-color:rgb(49, 49, 49);
}</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton  {
    background-color: rgb(255, 93, 93);
    color:white;
    border-style: outset;
    border-width: 5px;
    border-radius: 10px;
    border-color: rgb(255, 48, 48)
}

QPushButton:pressed {
     background-color:rgb(49, 49, 49);
     border-color:rgb(49, 49, 49);
}</string>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>541</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
   background-color: rgb(180, 180, 180);
   border-style: outset;
   border-width: 5px;
   border-radius: 10px;
   border-color: rgb(84, 84, 84)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>playBtn</zorder>
   <zorder>pauseBtn</zorder>
   <zorder>stopBtn</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
